website/src/views/home.js

```diff
@@ -38,6 +43,20 @@ function Home() {
             return () => mounted = false;
```

Comment:
This is a cleanup function, so you want to return this at the end of the `useEffect` lifecycle.  In other words, you want to wait until all promises have been resolved before returning this.  This can be a bit tricky, but you can do it with `Promise.all` I think.  You will prob have to refactor to be able to access all promises though in an iterable such that `Promise.all` can use it.

Reply:
The reason why the prior implementation was valid was because there was only one API call, so whenever we resolve the API call, we know we can return the cleanup function.  If you want, it isn't awful practice to make separate `useEffect` blocks for each API call that way you can resolve them all separately.  It also ensures parallel / pseudo-parallel requests.  Do whatever is easiest in your opinion.

Reply:
did not use a promise.all, instead moved if statement logic into the body of api.get. need clarification on multiple api calls and useeffect.